iam.tf

```diff
@@ -57,6 +57,7 @@ data "aws_iam_policy_document" "logs" {
     effect = "Allow"
 
     actions = [
+      "logs:CreateLogGroup",
```

Comment:
This one action should be only allowed if `!var.use_existing_cloudwatch_log_group`. Will you be able to update the code?

Reply:
@antonbabenko I'll have a look at it.